conf/httpd.conf.d/httpd.graphite

```diff
@@ -101,10 +101,10 @@ $KeepAliveTimeout = "10";
 $KeepAlive = "On";
 $MaxKeepAliveRequests = "100";
 
-$MaxClients = "56";
-$StartServers = "4";
-$MinSpareServers = "4";
-$MaxSpareServers = "20";
+$MaxClients = "2";
+$StartServers = "1";
+$MinSpareServers = "1";
+$MaxSpareServers = "1";
```

Comment:
I just remembered why I kept it low.
Since we have ~10 graphs on the PF dashboard, then additionnal httpd processes start whenever we access the status page on the admin since these requests are done at the same time. I agree we could put it a bit higher (like 4) but if we give him 10 processes it will use almost all of them.

Another solution would be to have graphite and httpd.admin share the same process pool and have them use VirtualHosts. If they fight together to process requests it has much less impact then if the portal and webservices where sharing the same processes.